src/main/kotlin/net/irisshaders/lilybot/extensions/util/ModUtilities.kt

```diff
@@ -128,13 +128,19 @@ class ModUtilities : Extension() {
 							icon = user.asUser().avatar?.url
 						}
 						timestamp = Clock.System.now()
-						color = DISCORD_BLACK
 						if (arguments.embed) {
+							if (arguments.color == DISCORD_BLURPLE) {
```

Comment:
Is this a typo? Not a fan of the blurple are we? XD

Reply:
This is so it matches current lily functionality with /say, which is to use discord_black (in the message logs), please say if there's a way to check that the colour argument just hasn't had anything added to it

unless, /say logs being changed to blurple by default is fine, then i can remove that

Reply:
I think we can disregard that part as the logging of the exact embed colour would be useful imo